build.sbt

```diff
@@ -13,7 +13,8 @@ resolvers += "Akka Snapshot Repository" at "http://repo.akka.io/snapshots/"
 libraryDependencies ++= Seq(
   "com.typesafe.akka" %% "akka-actor" % "2.4-M3",
   "com.typesafe.akka" %% "akka-testkit" % "2.4-M3",
-  "org.scalatest" % "scalatest_2.11" % "2.2.4" % "test"
+  "org.scalatest" % "scalatest_2.11" % "2.2.4" % "test",
+  "org.skife.com.typesafe.config" % "typesafe-config" % "0.3.0"
```

Comment:
Did I send this to you? 
I'd feel better using the official
libraryDependencies += "com.typesafe" % "config" % "1.3.0"